packages/framework/src/Model/Product/Elasticsearch/ProductExportRepository.php

```diff
@@ -182,7 +185,7 @@ protected function getExportedFieldValue(int $domainId, Product $product, string
             ProductExportFieldProvider::SEO_H1 => $product->getSeoH1($domainId),
             ProductExportFieldProvider::SEO_TITLE => $product->getSeoTitle($domainId),
             ProductExportFieldProvider::SEO_META_DESCRIPTION => $product->getSeoMetaDescription($domainId),
-            ProductExportFieldProvider::ACCESSORIES => $this->extractAccessoriesIds($product),
+            ProductExportFieldProvider::ACCESSORIES => $this->extractAccessoriesIds($product, $domainId, $this->productFrontendLimitProvider->getProductAccessoriesFrontendLimit()),
```

Comment:
framework bundle does not require the frontend api bundle in its composer.json file, so it's not possible to use frontend api classes in the framework bundle. 
I recommend not limiting exported fields at all.